app/uk/gov/hmrc/softdrinksindustrylevystub/services/SubscriptionGenerator.scala

```diff
@@ -21,27 +21,42 @@ import java.time.LocalDateTime
 import cats.implicits._
 import org.scalacheck._
 import org.scalacheck.support.cats._
-import uk.gov.hmrc.smartstub.Enumerable.instances.utrEnum
 import uk.gov.hmrc.smartstub._
 import uk.gov.hmrc.softdrinksindustrylevystub.models._
+import uk.gov.hmrc.smartstub.Enumerable.instances.utrEnum
 
 object SubscriptionGenerator {
 
   lazy val store: PersistentGen[String, CreateSubscriptionRequest] = genCreateSubscriptionRequest.asMutable[String]
 
   def genCreateSubscriptionRequest: Gen[CreateSubscriptionRequest] = {
     registrationGen                                          |@| // registration
-    Gen.choose(1,5).flatMap { n => Gen.listOfN(n, siteGen)}  |@| // sites
     Gen.choose(1,5).flatMap {
-      n => Gen.listOfN(n, entityActionGen)
+      n => Gen.listOfN(n, siteGen)}.sometimes                |@| // sites
+    Gen.choose(1,5).flatMap {
+      n => Gen.listOfN(n, entityActionGen).sometimes
     }                                                            // entityAction
   }.map(CreateSubscriptionRequest.apply)
 
   def genCreateSubscriptionResponse: Gen[CreateSubscriptionResponse] = {
-    Gen.const(LocalDateTime.now)                             |@| // processingDate
+    Gen.const(LocalDateTime.now).map {
+      x => x + "Z"
+    }                                                        |@| // processingDate
```

Comment:
Is this really the best way? can we not use a formatter?

Reply:
There really is. I struggled to find it but it is now LocalDateTime.now.atOffset(ZoneOffset.UTC)